mkosi/__init__.py

```diff
@@ -6687,42 +6673,27 @@ def load_args(args: argparse.Namespace) -> MkosiConfig:
         warn("The --boot-protocols is deprecated and has no effect anymore")
     delattr(args, "boot_protocols")
 
+    args.secure_boot_key = args.secure_boot_key or Path("./mkosi.secure-boot.key")
+    args.secure_boot_certificate = args.secure_boot_certificate or Path("./mkosi.secure-boot.crt")
```

Comment:
I think this can safely go into a `default=` on the option. I don't remember why I didn't put it there in the first place.